### PR TITLE
Render buffer on drawn shape

### PR DESCRIPTION
**Pull request by alexabird**
depends on:
- https://github.com/connexta/geospatialdraw/pull/24

### CODE REVIEW

**Review by pvargas — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/drawing-and-display.tsx [hunk 1]
```diff
@@ -381,6 +382,9 @@ export const OpenlayersDrawings = ({
     null
   )
 
+  const [buffer, setBuffer] = useState<number>(0)
+  const [bufferUnit, setBufferUnit] = useState<string>('')
```
Comment (sgalla) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/drawing-and-display.tsx:386]: Should this just get initialized to `meters` or does it not really matter?

**Review by sgalla — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/package.json [hunk 2]
```diff
@@ -101,7 +101,7 @@
     "font-awesome": "4.7",
     "geo-convex-hull": "1.2.0",
     "geolib": "^3.2.1",
-    "geospatialdraw": "0.4.5",
+    "geospatialdraw": "file://opt/geospatialdraw/geospatialdraw-0.4.6.tgz",
```
Comment (jrnorth) [inline ui-frontend/packages/catalog-ui-search/package.json]: Reminder that this change should not be merged in.

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/drawing-and-display.tsx [hunk 3]
```diff
@@ -420,6 +424,25 @@ export const OpenlayersDrawings = ({
...
+      setBuffer(
+        drawingModel.attributes.polygonBufferWidth ||
+          drawingModel.attributes.lineWidth
+      )
+      setBufferUnit(
+        drawingModel.attributes.polygonBufferUnits ||
+          drawingModel.attributes.lineUnits
+      )
+    }
+  }, [drawingModel])
```
Comment (jrnorth) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/drawing-and-display.tsx]: :pencil2: Consider `useCallback` instead to make this a bit shorter.

**Review by jrnorth — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/package.json (hunk 2 above)
Comment (alexabird) [reply ui-frontend/packages/catalog-ui-search/package.json]: Will update to 0.4.6 once geospatial draw is published

**Comment by alexabird** [comment]
build now

**Pull request merged**